src/test/java/org/sqlite/ResultSetTest.java

```diff
@@ -154,4 +156,18 @@ public void testCloseStatement()
 
         assertTrue(resultSet.isClosed());
     }
+
+    @Test
+    public void testReturnsNonAsciiCodepoints()
```

Comment:
My changes here didn't break anything, I just noticed there wasn't a test for this.